oat-sa/extension-tao-publishing
# feat: add Simple Roster Webhook to Remote environment

Related to: https://oat-sa.atlassian.net/browse/LSI-1379

it is **DRAFT** because it has not merged yet dependency

depends on https://github.com/oat-sa/tao-core/pull/3766

How to test:
1. leave the webhook field empty. notice that no webhook was triggered after the successful publishing
2. create multiple webhooks triggered by `Remote Delivery Created Event`, select one in the `Remote environment` config, see that only the chosen one was triggered after the successful publishing

![image](https://user-images.githubusercontent.com/36041347/227215093-205d6006-552c-4981-ba18-e80c26567242.png)
